### ISSUE TITLE
Can't build MultiingualPages project on windows 10

### ISSUE THREAD

**Issue by PiWiBardy** [SharePoint/sp-dev-solutions#125]
#### Category
- [ ] Question
- [ X] Bug
- [ ] Enhancement

#### Expected or Desired Behavior
The project MultiingualPages  should package :)

#### Observed Behavior
Unable to package the solution: getting the error : 
Could not find Element Manifest: sharepoint\assets/clientsideinstance.xml

#### Steps to Reproduce
Clean / Build and package the solution through on Windows 10 : 

gulp clean && gulp bundle --ship && gulp package-solution --ship

Thank you !

**Comment by wobba** [comment]
@juliemturner any insights?

**Comment by wobba** [comment]
I believe the solution to this should be to not include the SharePoint folder in the repo, asn it's a build folder. @juliemturner?

**Comment by YannickRe** [comment]
@wobba The SharePoint folder is a build folder, except for the "assets" subfolder which contains the necessary xml files for an extension to be activated on a site.

@PiWiBardy I don't have an issue with building and packaging this solution, after removing the package-lock.json. I also had to make the adjustments explained below to package.json and do a clean npm install.

@juliemturner It might be best to adjust the package.json file from
```
    "@pnp/common": "^1.2.9",
    "@pnp/logging": "^1.2.9",
    "@pnp/odata": "^1.2.9",
    "@pnp/polyfill-ie11": "^1.0.1",
    "@pnp/sp": "^1.2.9",
```
to
```
    "@pnp/common": "1.2.9",
    "@pnp/logging": "1.2.9",
    "@pnp/odata": "1.2.9",
    "@pnp/polyfill-ie11": "1.0.1",
    "@pnp/sp": "1.2.9",
```
I couldn't get npm install working with the existing package-lock.json and when I removed it, npm install loads the latest version of the mentioned packages (currently 1.3.2).  That version breaks the solution at compile time with some typescript errors (checkin is not defined on ClientSidePage or something).
